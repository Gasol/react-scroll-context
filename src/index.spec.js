import * as index from '.';

describe('@foo-software/react-scroll-context package', () => {
  it('should match snapshot', () => {
    expect(index).toMatchSnapshot();
  });
});
