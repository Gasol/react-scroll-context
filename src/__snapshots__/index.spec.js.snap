// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@foo-software/react-scroll-context package should match snapshot 1`] = `
Object {
  "ScrollProvider": [Function],
}
`;
